enterprise/models.py

```diff
@@ -808,6 +831,40 @@ def consent_needed(self):
         enterprise_customer = self.enterprise_customer_user.enterprise_customer
         return enterprise_customer.enforces_data_sharing_consent(EnterpriseCustomer.AT_ENROLLMENT)
 
+    @property
+    def audit_reporting_disabled(self):
+        """
+        Specify whether audit track data reporting is disabled for this enrollment.
+
+        * If the enterprise customer associated with this enrollment enables audit track data reporting,
+          simply return False.
+        * If the enterprise customer associated with this enrollment does not enable audit track data reporting,
+          return True if we are dealing with an audit enrollment, and False otherwise.
+
+        :return: True if audit track data reporting is disabled, False otherwise.
+        """
+        if not self.enterprise_customer_user.enterprise_customer.enables_audit_data_reporting:
+            return self.is_audit_enrollment
+
+        # Since audit data reporting is enabled, we always return False here.
+        return False
+
+    @property
+    def is_audit_enrollment(self):
+        """
+        Specify whether the course enrollment associated with this ``EnterpriseCourseEnrollment`` is in audit mode.
+
+        :return: Whether the course enrollment mode is of an audit type.
+        """
+        course_enrollment_api = EnrollmentApiClient()
```

Comment:
See https://github.com/edx/edx-enterprise/pull/156/files/8ead4196c7c632a82494f0ee268b25c38f53ccab#diff-6f7edc719ace33fecde94be26155c9c9R151 for where/how it previously was. Sorry for the comment spam in that diff!